src/harness/client.ts

```diff
@@ -392,6 +392,10 @@ namespace ts.server {
             });
         }
 
+        getProgramDiagnostics(): Diagnostic[] {
+            return this.getCompilerOptionsDiagnostics();
+        }
+

```

Comment:
This is the only thing which isn't JSDoc improvements, it fixed a TODO item I agreed with:

```
        // TODO: Rename this to getProgramDiagnostics to better indicate that these are any		         /**
         // diagnostics present for the program level, and not just 'options' diagnostics.		
```